x-pack/solutions/search/plugins/search_getting_started/public/components/tutorials/console_tutorials_group.tsx

```diff
@@ -27,73 +24,81 @@ interface TutorialMetadata {
 
 export const ConsoleTutorialsGroup = () => {
   const { application, console: consolePlugin, share } = useKibana().services;
-  const tutorials: TutorialMetadata[] = [
-    {
-      title: i18n.translate('xpack.searchGettingStarted.consoleTutorials.basicsTitle', {
-        defaultMessage: 'Search basics',
-      }),
-      dataTestSubj: 'console_tutorials_search_basics',
-      description: i18n.translate('xpack.searchGettingStarted.consoleTutorials.basicsDescription', {
-        defaultMessage: 'Learn how to create an index, add documents, and basic search techniques.',
-      }),
-      request: consoleTutorials.basics,
-      image: searchWindowIllustration,
-      buttonRef: useRef<HTMLButtonElement>(null),
-    },
-    {
-      title: i18n.translate('xpack.searchGettingStarted.consoleTutorials.semanticTitle', {
-        defaultMessage: 'Intro to semantic search',
-      }),
-      dataTestSubj: 'console_tutorials_semantic_search',
-      description: i18n.translate(
-        'xpack.searchGettingStarted.consoleTutorials.semanticDescription',
-        {
+  const assetBasePath = useAssetBasePath();
+  const tutorials: TutorialMetadata[] = useMemo(
```

Comment:
wondering, do we need to use `useMemo` ? `assetBasePath` may not change on render?  
we could move this definition outside `ConsoleTutorialsGroup` and keep a static array of objects like it was before?